qos/evm/endpoint.go

```diff
@@ -6,15 +6,26 @@ import (
 	"strconv"
 
 	qosobservations "github.com/buildwithgrove/path/observation/qos"
+	"github.com/buildwithgrove/path/protocol"
 )
 
 // The errors below list all the possible validation errors on an endpoint.
 var (
-	errNoChainIDObs             = fmt.Errorf("endpoint has not had an observation of its response to a %q request", methodChainID)
-	errInvalidChainIDObs        = fmt.Errorf("endpoint returned an invalid response to a %q request", methodChainID)
-	errNoBlockNumberObs         = fmt.Errorf("endpoint has not had an observation of its response to a %q request", methodBlockNumber)
-	errInvalidBlockNumberObs    = fmt.Errorf("endpoint returned an invalid response to a %q request", methodBlockNumber)
+	// empty response errors
 	errHasReturnedEmptyResponse = errors.New("endpoint is invalid: history of empty responses")
+
+	// chainID check errors
+	errNoChainIDObs      = fmt.Errorf("endpoint has not had an observation of its response to a %q request", methodChainID)
```

Comment:
Seems like we could move a few things to a shared string like `errNoObservationStr` and `errInvalidObsStr` and reuse it.